src/components/settings-modal/settings-modal.scss

```diff
@@ -2,13 +2,17 @@
 @use '../../styles/mixins' as mixins;
 
 .kui-theme--light {
-  --color-modal-header-text: #{colors.$black-700};
+  --color-modal-header-text: #{colors.$black-0};
   --color-upgrade-reminder-text: #{colors.$blue-300};
+  --color-modal-table-text: rgb(0 0 0 / 85%);
+  --color-modal-table-description: rgb(0 0 0 / 70%);
```

Comment:
You could check for rgb(...) in `_variables.scss` file instead of using rgb(...) color directly.

Reply:
Is there a way to get it with that alpha value?

`rgb(0 0 0)` is `$black-900 ` but what would be the equivalent for `rgb(0 0 0 / 85%)`?

Reply:
you can use this tool to convert rgb to hexcode https://rgbacolorpicker.com/rgba-to-hex, but worth confirming with @stephkaiser if the colours are correct after you convert them

Reply:
@SajidAlamQB I compared `$black-600: #323232` with` rgb(0 0 0 / 80%)` by mistake. So `#323232` and `#000000d9 ` are not same.

Reply:
great point @Huongg, I should have followed the original kedro colour palette instead, I've updated the designs in figma now so the colours don't use opacity. Hope the below helps

For dark UI:
- Settings main title, and setting names - `white-0` or #FFFFFF 100%
- Description - `white-900` or #C6CACE 100%
- Version text - `black-300` or #636363 100%
- Disabled button - `black-300` or #636363 100%
- Default button - `white-0` or #FFFFFF 100%

For light UI:
- Settings main title, and setting names - `black-900` or #000000 100%
- Description - `black-500` or #424242 100%
- Version text - `black-0` or #989898 100%
- Disabled button - `black-0` or #989898 100%
- Default button - `black-900` or #000000 100%